### ISSUE TITLE
Cannot read property 'projectFilename' of undefined

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#7392]
Starting to get these when switching between files in my JS/TS workspace:

```
[Extension Host] Object {seq: 0, type: "response", command: "projectInfo", request_seq: 13, success: false…}command: "projectInfo"message: "Error processing request. Cannot read property 'projectFilename' of undefined"request_seq: 13seq: 0success: falsetype: "response"__proto__: Object
```

**Comment by dbaeumer** [comment]
@jrieken this is with the JS project status bar item.

**Comment by jrieken** [comment]
this is https://github.com/Microsoft/TypeScript/issues/8212 which will be fixed one we have adopted TypeScript 2.0
